client/grpc-web-node-http-transport/src/index.ts

```diff
@@ -18,6 +18,10 @@ class NodeHttp implements grpc.Transport {
   }
 
   sendMessage(msgBytes: Uint8Array) {
+    if (!this.options.methodDefinition.requestStream) {
```

Comment:
From what I understand we only care about checking if the `requestStream` is false but you mentioned `responseStream` in your comment too. Perhaps I am missing something?

Reply:
If you only want to set the content-length header for unary methods you will need to check that both `requestStream` and `responseStream` are false.

Reply:
Updated :)